sql/core/src/test/resources/sql-tests/inputs/pgSQL/with.sql

```diff
@@ -0,0 +1,1222 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- WITH
+-- https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/sql/with.sql
```

Comment:
PostgresSQL 12 Beta2 was released on 20th June.
- https://www.postgresql.org/about/news/1949/

Could you check `https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/with.sql`?

Reply:
Sure, https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/sql/with.sql and https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/with.sql are equal.

Reply:
Thank you!